modules/ztp-troubleshooting-factory-pipelines.adoc

```diff
@@ -0,0 +1,216 @@
+
+🌡️  Status
+
+STARTED       DURATION    STATUS
+3 hours ago   4 minutes   Failed
+
+Message
+
+"step-mirror-olm" exited with code 255 (image: "quay.io/ztpfw/pipeline@sha256:d86d567f0ee76efdd5ea168fac3cbd5e8e7e479ddcea0be6aaf9e890de9566b3"); for logs run: kubectl -n spoke-deployer logs deploy-ztp-hub-run-vgwz6-deploy-disconnected-registry-xqz-kltxr -c step-mirror-olm
+
+
+📨 Input Resources
+
+ No input resources
+
+📡 Output Resources
+
+ No output resources
+
+⚓ Params
+
+ NAME              VALUE
+ ∙ spokes-config   config:
+  OC_OCP_VERSION: '4.10.9'
+  OC_ACM_VERSION: '2.4'
+  OC_OCS_VERSION: '4.9'
+spokes:
+ ∙ kubeconfig            /root/.kcli/clusters/test-ci/auth/kubeconfig
+ ∙ ztp-container-image   quay.io/ztpfw/pipeline:latest
+ ∙ mock                  false
+
+📝 Results
+
+ No results
+
+📂 Workspaces
+
+ NAME    SUB PATH   WORKSPACE BINDING
+ ∙ ztp   ---        PersistentVolumeClaim (claimName=ztp-pvc)
+
+🦶 Steps
+
+ NAME                             STATUS
+ ∙ update-global-pullsecret       Error
+ ∙ deploy-disconnected-registry   Completed
+ ∙ mirror-ocp                     Completed
+ ∙ mirror-olm                     Error
+
+🚗 Sidecars
+
+No sidecars
+----
+
+. Debug a task execution from the container in the cluster as follows:
+
+.. Get all pods in the `spoke-deployer` namespace:
++
+[source,terminal]
+----
+$ oc get pod -n spoke-deployer
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                              READY   STATUS      RESTARTS   AGE
+deploy-ztp-hub-run-rwh4j-deploy-acm-k92kf-pod-85n7t               0/1     Completed   0          159m
+deploy-ztp-hub-run-rwh4j-deploy-disconnected-registry-8j9-rk469   0/4     Completed   0          3h2m
+deploy-ztp-hub-run-rwh4j-deploy-httpd-server-fw49r-pod-lhkxf      0/1     Completed   0          3h2m
+deploy-ztp-hub-run-rwh4j-deploy-hub-config-vmgf2-pod-cjg72        0/1     Completed   0          149m
+deploy-ztp-hub-run-rwh4j-deploy-icsp-hub-c7tg7-pod-ntmqp          0/1     Completed   0          149m
+deploy-ztp-hub-run-rwh4j-pre-flight-865p2-pod-6wmj4               0/1     Completed   0          3h3m
+deploy-ztp-spokes-run-sp8hm-deploy-icsp-spokes-pre-76thd--2pg7t   0/1     Completed   0          97m
+deploy-ztp-spokes-run-sp8hm-deploy-metallb-d7cnj-pod-rmbcg        0/1     Completed   0          94m
+deploy-ztp-spokes-run-sp8hm-deploy-ocs-k7hf9-pod-7rwwq            0/1     Completed   0          92m
+deploy-ztp-spokes-run-sp8hm-deploy-spokes-pmbnz-pod-kp5fc         0/2     Completed   0          123m
+deploy-ztp-spokes-run-sp8hm-pre-flight-zwdsn-pod-l2v7h            0/1     Completed   0          123m
+spoke-deploy-disconnected-registry-spokes-run-t6k2d-pod-cnm5t     4/4     NotReady    0          34s
+----
+
+.. Log in to the pod in `NotReady` state:
++
+[source,terminal]
+----
+$ oc debug pod/spoke-deploy-disconnected-registry-spokes-run-t6k2d-pod-cnm5t -n spoke-deployer
+Defaulting container name to step-deploy-disconnected-registry.
```

Comment:
Would you mind moving the output to an "example output" section like you do above (since we want to separate commands from output)?